.github/workflows/pushimage-next.yaml

```diff
@@ -39,12 +39,36 @@ jobs:
         repo: ['devfile/registry']
     runs-on: ubuntu-latest
     steps:
+      - name: Get the repository name and commit sha
+        id: get_commit_info
+        run: |
+          # Check if repository name is not set in the client payload
+          # If it's not set, the triggered commit came from this repo (registry-support)
+          # If it is set, the triggered commit came from registry-viewer
+          if [ -z ${{ github.event.client_payload.repo }} ];
+          then
+            echo ::set-output name=repo::$(echo ${{ github.event.repository.name }})
+          else
+            echo ::set-output name=repo::$(echo ${{ github.event.client_payload.repo }})
```

Comment:
I'm assuming `client_payload` is the github context syntax and this is the https://github.com/devfile/registry-viewer/blob/main/.github/workflows/push-changes.yaml#L20 dispatch action syntax?

Reply:
Yup.